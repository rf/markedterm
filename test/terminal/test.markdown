this is some *bold text*

# derp

```
indentation
```
