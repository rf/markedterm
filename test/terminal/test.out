
      this is some [4m[1mbold text[22m[24m

   [4m[34m[1mderp[22m[39m[24m

[90m         [90mindentation[39m[39m

